tests/API/XCCDF/unittests/test_fix_arf.sh

```diff
@@ -12,6 +12,7 @@ ansible_task1a="\- name: ensure everything passes"
 ansible_task1b="shell: /bin/true"
 ansible_task2a="\- name: correct the failing case"
 ansible_task2b="shell: /bin/false"
+regex="^$|[\#]$|^([\#][[:blank:]][^\-])"
```

Comment:
I highly recommend commenting what regexes do, regexes are hard to read even for people with a ton of experience with them